driver/panda_stream.c

```diff
@@ -190,10 +196,13 @@ static irqreturn_t stream_isr(int irq, void *dev_id)
     void *reg_base = open->pcap->reg_base;
 
     uint32_t status = readl(reg_base + PCAP_IRQ_STATUS);
-    if (verbose)
```

Comment:
Is `verbose` redundant now?  If so, we probably need to remove it from the module parameters.

Reply:
done